test/SymbolGraph/ClangImporter/ExportedImport.swift

```diff
@@ -0,0 +1,12 @@
+// RUN: %empty-directory(%t)
+// RUN: cp -r %S/Inputs/ExportedImport/ObjcProperty.framework %t
+// RUN: %target-swift-frontend(mock-sdk: %clang-importer-sdk) -enable-objc-interop -emit-module -o %t/ObjcProperty.framework/Modules/ObjcProperty.swiftmodule/%target-swiftmodule-name -import-underlying-module -F %t -module-name ObjcProperty %S/Inputs/ExportedImport/A.swift
```

Comment:
Is it worth also having a separate test for `swift-symbolgraph-extract`?

Reply:
`swift-symbolgraph-extract` doesn't have support for processing re-exports (https://github.com/apple/swift/issues/59920) so any tests i add for that will just confirm that the symbols don't exist.